components/log-viewer-webui/client/src/components/ResultsTimeline/index.tsx

```
@@ -0,0 +1,284 @@
+                        enabled: false === isInputDisabled,
+                        backgroundColor: "rgba(64,150,160,0.3)",
+                    },
+                    mode: "x",
+                    onZoom: ({chart}) => {
+                        const xAxis = chart.scales["x"];
+                        if ("undefined" === typeof xAxis) {
+                            return;
+                        }
+                        const {min, max} = xAxis;
+                        const newTimeRange = {
+                            begin: convertZoomTimestampToUtcDatetime(parseInt(min, 10)),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)